reactioncommerce/reaction-admin
# chore: adding git hooks for commit-msg

Resolves #345
Impact: **minor**
Type: **chore**

## Issue
No commit-msg hooks, so the commit message were a little out of consistency.

## Solution
Adding commit-msg hooks using husky.

## Breaking changes
None

## Testing
1. Pull the branch. 
1. Run `npm i`
1. Make some edit.
1. Try to commit, something weird like "this is a invalid commit". The hooks should fail.
2. Fix the messge to something like "fix: valid commit", you should see the commit succeed.
